app/src/main/java/com/android/voyageur/model/user/UserViewModel.kt

```diff
@@ -9,6 +9,7 @@ import com.google.firebase.auth.FirebaseUser
 import kotlinx.coroutines.Job
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.launch
 
 open class UserViewModel(
```

Comment:
Documentation is missing in this class. I believe we should add it especially because it is a ViewModel and other colleagues might need to use it without having background on how it works. Could you add brief descriptions for each method and the class as a whole?

Reply:
Thank you for the suggestion. I agree that it’s essential, especially for a ViewModel that others may use without in-depth knowledge of its functionality. To improve both usability and maintainability, I’ve added brief descriptions for the class and each method to enhance clarity for our team.